src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -215,6 +222,13 @@ public boolean hasParserResults() {
             automaticallySetFileLinks(loaded);
         }
 
+        if (cli.isWriteXMPtoPdf()) {
+            System.out.println("writing for " + cli.getWriteXMPtoPdf());
+            if (!loaded.isEmpty()) {
+                writeXMPtoPdf(loaded, cli.getWriteXMPtoPdf(), Globals.prefs);
```

Comment:
I know, in the CLI extraction of Globals is far from done. Could you please take a quick look, of it does make any sense to try to extract Globals or would this blow this PR too much up?

Reply:
The plan was to try it out using Globals and removing it later, but the removal is harder than I thought. There are lots of dependencies that need to be fixed.
I was working on implementing unit tests this afternoon and noticed that the dependency on globals is a problem there as well (because Globals.prefs is null when running them and initializing it is forbidden).

Reply:
The part I am most struggling with is the MessageHandler. I need to extend it's interface so I can pass a preferences object, but that breaks many tests where I don't have a preferences object that I can pass

Reply:
Then forget about Globals extraction for and concentrate on integrating the feature. This is more important now. We need sometime in the future a solution, but not right now for this.

Reply:
The thing is if I keep using Globals.prefs, I cannot implement any tests, as the read-access to Globals.prefs fails since Globals.prefs is null.